iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/setup/ProvisioningCommon.java

```diff
@@ -239,7 +239,6 @@ public void tearDown()
 
         if (testInstance != null && testInstance.securityProvider != null && testInstance.securityProvider instanceof SecurityProviderTPMEmulator)
         {
-            testInstance.provisioningServiceClient = null;
```

Comment:
Fixing a small test bug as well. No need to null this client when it is still used below for cleanup